plugins/aws-appmesh/plugin/commands.go

```diff
@@ -208,27 +206,70 @@ func (plugin *Plugin) setupIptablesRules(
 	return nil
 }
 
+// setupIngressRules installs iptable rules to handle ingress traffic.
+func (plugin *Plugin) setupIngressRules(iptable *iptables.IPTables,
+	config *config.NetConfig) error {
+	if config.ProxyIngressPort == "" {
```

Comment:
I understand you are checking for this misconfig case in netconfig validation, but do you think we should return if either one of ProxyIngressPort and AppPorts is empty here, just to be safe? If for whatever reason AppPorts is empty but ProxyIngressPort is not, line 222 will fail with a cryptic error.

Reply:
Agree, added OR to catch empty AppPorts here.